src/TesterInternal/GeoClusterTests/TestingClusterHost.cs

```diff
@@ -0,0 +1,251 @@
+﻿using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Orleans.TestingHost;
+using System.Reflection;
+using System.Globalization;
+using UnitTests.Tester;
+using Orleans.Runtime.Configuration;
+using System.Net;
+using System.Net.Sockets;
+using Orleans;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Tests.GeoClusterTests
+{
+    /// <summary>
+    /// A utility class for tests that include multiple clusters.
+    /// It calls static methods on TestingSiloHost for starting and stopping silos.
+    /// </summary>
+    public class TestingClusterHost   
```

Comment:
why TestingClusterHost    has to use TestingSiloHost as a static object? Why can't it inherit it, or use as an instance object?